README.md

```diff
@@ -0,0 +1,229 @@
+# aquasurf
+Efficient Activation Function Optimization through Surrogate Modeling
+
```

Comment:
Add a link to your paper?

Reply:
Will add one!  I haven't uploaded it to arxiv yet because we needed to add the repo link to the paper.

Reply:
Git it. 🐔 + 🥚